cmd/entrypoint/main.go

```diff
@@ -43,6 +43,13 @@ var (
 	results             = flag.String("results", "", "If specified, list of file names that might contain task results")
 	timeout             = flag.Duration("timeout", time.Duration(0), "If specified, sets timeout for step")
 	breakpointOnFailure = flag.Bool("breakpoint_on_failure", false, "If specified, expect steps to not skip on failure")
+	onError             = flag.String("on_error", "", "Set to \"continue\" to ignore an error and "+
+		"continue when a container terminates with a non-zero exit code. "+
+		"Set to \"fail\" to declare a failure with a step error and stop executing the rest of the steps.")
+	stepPath = flag.String("step_path", "", "Relative step path, creates the specified path under /tekton/steps/ which "+
+		"can be used to store the step metadata e.g. <step-name> in /tekton/steps/<step-name>/")
+	stepPathLink = flag.String("step_path_link", "", "Relative step path, creates a symbolic link to the "+
+		"specified step path e.g. <step-index> in /tekton/steps/<step-index>")
```

Comment:
Three optional flags here introduced here:

* `onError` - this flag is initialized to either `continue` or `fail` based on the step definition
* `stepPath` - the relative path specified in this flag is created under /tekton/steps/
* `stepPathLink` - the relative path specified in this flag is created as a symbolic link to `stepPath`

Reply:
What's the reasoning about `stepPath` and `stepPathLink` ? (aka are they required for this feature, if yes, how/why?)